lib/board.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require_relative 'figures/knight'
+require_relative 'figures/rook'
+require_relative 'figures/bishop'
+require_relative 'figures/king'
+require_relative 'figures/queen'
+require_relative 'figures/pawn'
+
+class Board
+  attr_reader :figures
+
+  def initialize
```

Comment:
Alright, so this data format, I can live with. 

I don't like, that your data is inconsistent: You've changed the user interface to work with classical board signatures (a1, a2 etc), but internal logic still seems to work wholly on your personal board marks. It's rather unclear to me, even more so given your methods are not documented.

I would suggest that you continue, and reorganise the rest of your logic to be based on classical signatures.

Also, why the `prepare_board` method? Can you just do all of the logic in the `initialize` ?